api/src/main/java/io/opentelemetry/trace/Tracer.java

```diff
@@ -73,7 +75,31 @@
    *     from the Context.
    * @since 0.1.0
    */
-  Span getCurrentSpan();
+  default Span getCurrentSpan() {
+    return TracingContextUtils.getSpan(Context.current());
+  }
+
+  /**
+   * Gets the current Span from the specified Context.
+   *
+   * <p>To install a {@link Span} to the specified Context use {@link #withSpan(Span)}.
+   *
+   * <p>startSpan methods do NOT modify the current Context {@code Span}.
+   *
+   * @param context The {@code Context} to get the current Span from.
+   * @return a default {@code Span} that does nothing and has an invalid {@link SpanContext} if no
+   *     {@code Span} is associated with the current Context, otherwise the current {@code Span}
+   *     from the Context.
+   * @since 0.1.0
+   */
+  default Span getCurrentSpan(Context context) {
+    return TracingContextUtils.getSpan(context);
+  }
```

Comment:
That was the alternative, yes. @mwear mentioned it feels better on the actual `Tracer` instance, but static methods would be an option.

Reply:
One general thought is I would expect the "job" of managing the active span to be in the `TracerProvider` - that is the concept of tracing in OpenTelemetry, so it needs to make sure the active span is managed properly. Having `Tracer` shortcut to it for convenience sounds nice though.

Also, for static I'd expect `Span.current` to be simpler than requiring a global tracer, it could shortcut to the global `TracerProvider`, and no need to introduce the "global tracer". That doesn't mean we may not want an anonymous tracer in the API for users to use in their app, that discussion has come up, but it should be orthogonal to context management.

Reply:
Given that `Span`s are handled by `Tracer`, I'm more inclined to put it there directly, instead of being a shortcut.

Making this a static set of members in `Tracer` (e.g. `Tracer.getCurrentSpan()`) sounds fine by me (if we decide to go this route).

Reply:
+1 for `Span.current()`, which nicely parallels `Context.current()`